simulation.py

```diff
@@ -143,12 +143,17 @@ def simulate(results_queue, data_all, alpha_settings_path, skip_evaluation):
     pnl_function = TRADING_CONFIGS[settings.target][PNL_FUNCTION](
         OFFICIAL_CURRENCY[region], TRADING_INSTRUMENTS[settings.target])
     results = Simulator(settings, data_required, pnl_function).run()
+    eval_url = 'http://127.0.0.1:5000/'
+    eval_data = {
```

Comment:
Since this object includes `indicator_names`, which is not really something to be evaluated but more like configurations, I think "eval_data" may not be a good name.
Maybe "eval_request"?